docs/source/quick_start.rst

```diff
@@ -6,7 +6,10 @@ may be running remotely.  With sufficient login credentials and an API Key (see
 Modelica models may be uploaded, compiled, and executed on a server.  The results can be either processed on the server
 with a custom function or downloaded to the client for further analysis.  
 
-A simple example of this functionality is::
+An analysis could be setup and executed using either the :ref:`FMU ID <1. FMU ID based workflow>` or a :ref:`Class name <2. Class name based workflow>` based workflow.

```

Comment:
I think "FMU ID" is a bit to implementation specific. I think it would be as clear and for some users less confusing if it just said "FMU" in all places.